tests/Feature/QuestionApiTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionApiTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /**
+     * A basic feature test example.
+     */
+    public function test_add_new_question_success(): void
+    {
+        $response = $this->postJson('/api/questions', [
+            'question' => 'Will this request work?',
+            'hint' => "Let's see",
+            'points' => 2,
+            'quiz_id' => 2,
```

Comment:
Once you add in the exists validation rule on the quiz_id, you'll find this fails. You'll need to use your Quiz factory to create a quiz before sending this test request.